src/lib/toastr/toast-noanimation.component.ts

```diff
@@ -19,7 +19,7 @@ import { ToastrService } from './toastr.service';
     <span aria-hidden="true">&times;</span>
   </button>
   <div *ngIf="title" [class]="options.titleClass" [attr.aria-label]="title">
-    {{ title }}
+    {{ title }} <ng-container *ngIf="options.countDuplicates && refCount > 1">[{{ refCount }}]</ng-container>

```

Comment:
What if, instead of storing `refCount` in the component, you store it in the `ToastPackage` and reference that directly in the component. Then when the `ToastrService` detects a duplicate, it can increment that value itself.

Then you wouldn't need a reference to `options.countDuplicates` and just need to do something like:

```ts
<ng-container *ngIf="toastPackage.refCount">[{{ refCount }}]</ng-container>
```

If refCount defaults to 0, it'll never show unless the value is incremented.

Just a suggestion. What do you think?

Reply:
That's a valid point. I now followed the example of the **preventDuplicates** and **resetTimeoutOnDuplicate** flags and moved the global check into the ToastrService.

Reply:
If I move the refCount variable to the toastPackage, I will have to reference the toastPackage in ActiveToast. Otherwise, how would I access it from the ToastrService?

I'm not 100% familiar with the project yet, I might be missing something.